src/styles/card.scss

```diff
@@ -0,0 +1,156 @@
+@import './variables';
+@import './mixins';
+
+.container {
+  padding: 50px 40px;
+
+  display: grid;
+  grid-template-columns: repeat(auto-fit, 200px);
+  gap: 48px 48px;
+  justify-content: center;
+
+  @include two-columns {
+    grid-template-columns: repeat(auto-fit, 200px);
```

Comment:
And here we have a problem, actually 2:
1. You are using **auto-fit** which auto calculates the column count based on element width. In this case, as the element is the whole page itself, and you are applying it for screens of max 488px, it ends up returning 2 columns (488 / 200 = 2,...). However, if let's say, you wanted 2 columns on a bigger screen, this would not be the case. And if a screen is 630px wide, it would result in 3 columns instead of 2, which is also not a desired behavior here.
2. Second point is that you are using the same declaration for all mixins that represent screen size, so either there is no need to use the mixins, or the style declaration should be different for all mixins. Can you guess which option is the correct one? :)

**There is a hint in this image:
![image](https://github.com/mate-academy/layout_catalog/assets/54222238/dff78f89-15b3-445e-a1e1-3aab7adf39f5)

Reply:
I just checked. It says I'm 1 commit behind, but just commited an update and now it's working great!